### PR TITLE
add ethereum signed packages

### PR DESCRIPTION
**Pull request by PhilippLgh**
This EIP specifies Ethereum Signed Packages which are container files such as .zip, .tar or .asar files containing application source code, assets, executables or smart contracts which are digitally signed using Ethereum's signature schemes.

**Discussions**
[https://ethereum-magicians.org/t/package-and-dependency-managers/2418](https://ethereum-magicians.org/t/package-and-dependency-managers/2418)
[https://github.com/PhilippLgh/ethereum-signed-packages](https://github.com/PhilippLgh/ethereum-signed-packages)

### CODE REVIEW

File: EIPS/eip-draft-signed-packages.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+eip: <to be assigned>
+title: Ethereum Signed Packages
+author: Philipp Langhans (@PhilippLgh) <philipp@ethereum.org>
+discussions-to: https://github.com/PhilippLgh/ethereum-signed-packages/issues
+status: Draft
+type: Standards Track (Core, Networking, Interface, ERC)
+category (*only required for Standard Track): ERC
```
Comment (nicksavers) [inline EIPS/eip-draft-signed-packages.md]: You can remove the optional lines if you don't use them.

File: EIPS/eip-draft-signed-packages.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+created: 2019-02-27
+requires (*optional):
```
Comment (nicksavers) [inline EIPS/eip-draft-signed-packages.md]: You can remove the optional lines if you don't use them.

**Comment by gaga11111** [comment]
pleace i need more info

‫في الجمعة، 8 مارس 2019 في 11:27 ص تمت كتابة ما يلي بواسطة ‪Nick Savers‬‏ <‪
notifications@github.com‬‏>:‬

File: EIPS/eip-draft-signed-packages.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+eip: <to be assigned>
```
Comment (axic) [inline EIPS/eip-draft-signed-packages.md]: Please use 1781 here.

File: EIPS/eip-draft-signed-packages.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- helps to decouple development from distribution and hosting
+- compatible to emerging blockchain based identity provider solutions
+- leverages existing and emerging ecosystem tools for key management such as keybase or metamask to manage keys and sign packages in the browser
+- certificate revocation lists can be implemented on the blockchain to be tamper proof and "instantly" updated to prevent bad actors from using stolen keys to sign malware
+
+Discussion on the magicians forum: [Package and Dependency Managers](https://ethereum-magicians.org/t/package-and-dependency-managers/2418)
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+A specification draft and reference implementation can be found at:
```
Comment (axic) [inline EIPS/eip-draft-signed-packages.md]: I think the key point of ERCs is to include actual specification. Would it be possible to include an excerpt here at least or to point to a specification commit hash?

File: EIPS/eip-draft-signed-packages.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
+---
+eip: <to be assigned>
+title: Ethereum Signed Packages
+author: Philipp Langhans (@PhilippLgh) <philipp@ethereum.org>
+discussions-to: https://github.com/PhilippLgh/ethereum-signed-packages/issues
+status: Draft
+type: Standards Track (Core, Networking, Interface, ERC)
```
Comment (axic) [inline EIPS/eip-draft-signed-packages.md]: This should only have "Standards Track"

**Comment by axic** [comment]
Ping @PhilippLgh

**Comment by PhilippLgh** [comment]
thanks @axic - tried to integrate all suggested improvements. :+1:

**Comment by axic** [comment]
It has two spelling issues:
```
EIPS/eip-draft-signed-packages.md:28: CAs  ==> case
EIPS/eip-draft-signed-packages.md:28: dependant  ==> dependent
```

Also the file should be named `eip-1781.md`

**Comment by PhilippLgh** [comment]
thanks @axic I can correct the second spelling problem. the first is a false positive in the spell checker. should I just remove it then?

**Comment by axic** [comment]
> the first is a false positive in the spell checker. should I just remove it then?

You could just change it to "certificate authorities".

File: EIPS/eip-1781.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
+---
+eip: 1781
+title: Ethereum Signed Packages
```
Comment (axic) [inline EIPS/eip-1781.md]: You use the shorthand EPK below, perhaps include it here?

File: EIPS/eip-1781.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+discussions-to: https://github.com/PhilippLgh/ethereum-signed-packages/issues
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-02-27
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+# Ethereum Signed Packages (EPK)
```
Comment (axic) [inline EIPS/eip-1781.md]: This seems to be an invalid heading (all headings are 2 or more indented).

**Pull request closed without merging**